libethereum/ChainParams.cpp

```diff
@@ -37,6 +59,12 @@ ChainParams::ChainParams(string const& _json, h256 const& _stateRoot)
     *this = loadConfig(_json, _stateRoot);
 }
 
+ChainParams::ChainParams(std::string const& _s, AdditionalEIPs const& _additionalEIPs)
```

Comment:
(Nit) Can we choose a name for the `_s `arg which reflects the semantics of the data it contains?